### PR TITLE
Bolt Bend can never target Seeds of Strength

### LINKED ISSUE #2299: Something about Thirst for Knowledge and similar cards.
Hello.  If I want to discard two artifact cards for discard effect of "Thirst for knowledge"(or two basic land cards for Thirst for discover), what shall I do? Thanks for your efforts.

### PR DESCRIPTION
**Pull request by tool4ever**
- > If there are no legal targets that could be selected, the targets remain unchanged. Note that this may not mean that those targets are illegal as the spell resolves. For example, if an opponent in a two-player game has hexproof and casts Lava Spike targeting you, Chef's Kiss will not be able to change the target. However, this doesn't make you an illegal target, and both the original spell and the copy will still target you and deal 3 damage to you. 

- Closes #2299

### CODE REVIEW

**Review by Agetian — APPROVED** [review]

File: forge-gui/res/cardsfolder/c/captured_by_the_consulate.txt [hunk 1]
```diff
@@ -4,6 +4,6 @@ Types:Enchantment Aura
 K:Enchant creature you don't control
 A:SP$ Attach | Cost$ 3 W | ValidTgts$ Creature.YouDontCtrl | AILogic$ Curse
 S:Mode$ Continuous | Affected$ Creature.EnchantedBy | AddHiddenKeyword$ CARDNAME can't attack. | Description$ Enchanted creature can't attack.
-T:Mode$ SpellCast | ValidCard$ Card | ValidActivatingPlayer$ Opponent | IsSingleTarget$ True | TriggerZones$ Battlefield | Execute$ TrigChangeTarget | TriggerDescription$ Whenever an opponent casts a spell, if it has a single target, change the target to enchanted creature if able.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/c/captured_by_the_consulate.txt:7]: @tool4ever does this mean we should remove the Parameter for "IsSingleTarget" ?

Comment (Hanmac) [reply forge-gui/res/cardsfolder/c/captured_by_the_consulate.txt:7]: it is written as Intervening IF Clause

if for example the Opponent cast as Spell that has two or more "target" but each of them target the same target this Aura would trigger.

But if now one of the two or more targets is moved with something like Spellskite while leaving the other targets in tact.(?)
Then when this trigger tries to resolve, it should check the IF Clause again, see that the Spell now has more than a single target and should fizzle.

Comment (tool4ever) [reply forge-gui/res/cardsfolder/c/captured_by_the_consulate.txt:7]: No, check out CR 115.9a

the wordings there are really easy to mix up

this case counts each object

this is also enforced by this ruling as reminder:
> If a spell has multiple targets, it doesn't have a single target even if the same object is chosen for each of those targets.

Comment (Hanmac) [reply forge-gui/res/cardsfolder/c/captured_by_the_consulate.txt:7]: Also this Ruling:
> If a spell or ability targets the same player or object multiple times, you can’t target it with Bolt Bend. (2019-05-03)

**Pull request merged**